gitlab/server/service.sls

```diff
@@ -0,0 +1,43 @@
+{%- from "gitlab/map.jinja" import server with context %}
+{%- if server.enabled %}
+
+include:
+{%- if server.source.engine == 'pkg' %}
+- gitlab.server._service_pkg
+{%- else %}
+- gitlab.server._service_src
+{%- endif %}
+
+{%- if server.initial_data is defined %}
+
+gitlab_backup_dirs:
+  file.directory:
+  - names:
+    - /root/gitlab/scripts
```

Comment:
Don't abuse root's home directory for backups, etc. I would instead use /var/cache/salt/minion or /var/backups or some better location?